doc/source/options.rst

```diff
@@ -28,7 +28,7 @@ You can get/set options directly as attributes of the top-level ``options`` attr
    pd.options.display.max_rows = 999
    pd.options.display.max_rows
 
-There is also an API composed of 5 relevant functions, available directly from the ``pandas``
+The API is composed of 5 relevant functions, available directly from the ``pandas``
```

Comment:
The "there is also" does give a bit more the notion of "next to the possibility of directly getting/setting options on`pd.options. ..`, you also have some functions to work with them". So maybe "Additionally, the ..", or "the api is further composed ..." ?